addons/calendar/models/calendar_recurrence.py

```diff
@@ -444,7 +444,7 @@ def _get_occurrences(self, dtstart):
         dtstart = pytz.utc.localize(dtstart).astimezone(timezone)
         # dtstart is given as a naive datetime, but it actually represents a timezoned datetime
         # (rrule package expects a naive datetime)
-        occurences = self._get_rrule(dtstart=dtstart.replace(tzinfo=None))
+        occurences = self._get_rrule(dtstart=dtstart)
```

Comment:
if `.replace(tzinfo=None)` is removed, I think the comment makes no sense and should also be removed :wink: